lib/fastlane/setup/setup_ios.rb

```diff
@@ -52,6 +45,41 @@ def run
       # rubocop:enable Lint/RescueException
     end
 
+    def default_setup(path: nil)
+      copy_existing_files
+      generate_appfile(manually: false)
+      detect_installed_tools # after copying the existing files
+      create_app_if_necessary
+      enable_deliver
+      FileUtils.mkdir(path)
+      generate_fastfile(manually: false)
+      show_analytics
+    end
+
+    def manual_setup(path: nil)
+      copy_existing_files
+      generate_appfile(manually: true)
+      detect_installed_tools # after copying the existing files
+      ask_to_enable_other_tools
+      FileUtils.mkdir(path)
+      generate_fastfile(manually: true)
+      show_analytics
+    end
+
+    def ask_to_enable_other_tools
+      unless self.itc_ref && self.portal_ref
```

Comment:
That's super confusing and might even be wrong. How is the priority of `unless` and `&&`